packages/runtime/src/index.ts

```diff
@@ -154,3 +164,32 @@ export default function init(options: RuntimeOptions = {}) {
   window.addEventListener('load', execute)
   window.addEventListener('DOMContentLoaded', execute)
 }
+
+function autoPrefixer(): Postprocessor {
```

Comment:
That could be a very complex thing to do - are you really sure? 👀

If so, I think we should have a separate package like `@unocss/autoprefixer` and add unit tests for it

Reply:
I can see the benefit of autoprefixer, but I only do this in runtime because I'm underqualified to do package & bundling.

I tracked the use of this function in vue and so far I can only find it and its usage inside `runtime-dom`.
https://github.com/vuejs/vue-next/blob/92f11d6740929f5b591740e30ae5fba50940ec82/packages/runtime-dom/src/modules/style.ts#L69-L70

Also the related commit by Evan himself
https://github.com/vuejs/vue-next/commit/2b2727e62cf791d301dc254628d965a69bf3398a

Reply:
Ok, at least we need to have this function in a separate file and test it via direct imports